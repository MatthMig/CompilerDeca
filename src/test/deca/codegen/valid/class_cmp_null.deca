//  Description : 
// 
//      Test incorrect qui fait une comparaison avec null
// 
//  Résultats :
// 
//      N/A 
// 
//  Historique : 
// 
//      Créé le 13/06 par gl03

class A {

}

{
    A oA = new A();
    if (oA==null){
        println("vous n'allez pas arrivé ici");
    }
    else {
        println("Le test fonctionne pour l'instant");
    }
    oA = null; 
    if (oA!=null){
        println("vous n'allez pas arrivé ici");
    }
    else {
        println ("Le test fonctionne jusquau bout");
    }
}