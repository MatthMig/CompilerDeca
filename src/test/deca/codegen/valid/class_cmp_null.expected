Le test fonctionne pour l'instant
Le test fonctionne jusquau bout